content/doc/developer/development-environment/ide-configuration.adoc

```diff
@@ -22,6 +22,57 @@ This plugin has the following features:
 == Netbeans
 http://plugins.netbeans.org/plugin/43938/?show=true
 
+== Visual Studio Code
+See: https://code.visualstudio.com/docs/languages/java for useful extensions
+
+Create the following 2 files in your project:
+
+`~/.vscode/tasks.json`:
+```
```

Comment:
this is asciidoc, not markdown,

https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#why-asciidoc

You'll want something like:
```suggestion
[source, json]
----
```